src/main/java/frc/robot/subsystems/internals/InternalSubsystem.java

```diff
@@ -215,18 +227,18 @@ public void periodic() {
 
             // Reset states
             // If the only ball in the system is the one we just shot, mark the shot as completed
-            if (ballCount <= 1) shotRequested = false;
+            if (ballCount < 1) shotRequested = false;
             rejectingChecked = false;
```

Comment:
This is a little more complicated. If we skip the `stagingExitBall` timer and don't wait the 0.5 seconds, rejection logic won't check the color of the second ball. If the two balls are of different colors, it will shoot both either rejecting or shooting (which is bad in both cases).

Reply:
Perhaps a solution is to only skip the `stagingExitBall` check if the balls are of the same color? Might be a bit messy to implement